src/tests/databases/core_9/meta.txt

```diff
@@ -14,11 +14,12 @@
 2	1	species.taxonomy_id	6239
 10	1	species.type	monsters
 5	1	species.url	Jabbe
-17	1	genebuild.version	EXT01
+17	1	genebuild.version	ENS01
```

Comment:
Don't get why this PR change so much test data.

Reply:
We had 12 assert statements commented out. They pass now.